### ISSUE TITLE
Must be hosted in the cloud. 

### ISSUE THREAD

**Issue by jcpv9824** [jcpv9824/agile-final-project#9]
**As a** System Administrator
 **I need** the service to be hosted in the cloud
 **So that** we can ensure scalability, reliability, and accessibility of the service.
   
 ### Details and Assumptions
 * The service must be hosted on a cloud platform (e.g., AWS, Azure, Google Cloud).
 * The hosting solution should support automatic scaling based on load.
 * Proper security measures should be in place to protect the data and the service.
   
 ### Acceptance Criteria  
   
 ```gherkin
    Given the service is deployed in the cloud
    When users access the service at any time from any location
    Then they should be able to use the service without any accessibility or performance issues
 ```
